episodes/multi-dimensional-images.md

```diff
@@ -0,0 +1,449 @@
+allows switching channels - try moving both sliders and see how the image 
+display changes.
+
+We can check the image's dimensions with:
+
+```python
+
+image = viewer.layers["membrane"].data
+print(image.shape)
+```
+
+```output
+# (c, z, y, x)
+(2, 60, 256, 256)
+```
+
+This clearly shows all 4 dimensions, with dimension 0 as channels and dimension 
+1 as z. This matches the labels at the left hand side of each of the two 
+sliders. We can separate the channels again by right clicking on the 'membrane' 
+image layer and selecting:  
+`Split Stack`
+
+Given its 4 dimensions, this image is stored in a 4D image array. This is 
+similar to having one 3D image array per channel, stacked side by side:
+
+![](fig/3d-4d-arrays.png){alt="A diagram comparing 3D and 4D 
+image arrays" width='80%'}
+
+:::::::::::::::::::::::::::::::::::::: callout
+
+## Reading channels in Napari
+
+When we open the cells image through the Napari menu, it is really calling 
+something like:
+
+```
+from skimage import data
+viewer.add_image(data.cells3d(), channel_axis=1)
+```
+This adds the cells 3D image (which is stored as zcyx), and specifies that 
+dimension 1 is the channel axis. This allows Napari to split the channels 
+automatically into different layers.
+
+Usually when loading your own images into Napari e.g. with the 
+napari-aicsimageio plugin (as we will see in the next episode), the channel 
+axis should be recognised automatically. If not, you may need to add the image 
+via the console as above, manually stating the channel axis.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+
+## Time
+
+You should have already downloaded the MitoCheck dataset as part of the 
+[setup instructions](../learners/setup.md) - if not, you can download it by 
+clicking on '00001_01.ome.tiff' on [this page of the OME website
+](https://docs.openmicroscopy.org/ome-model/5.6.3/ome-tiff/data.html#mitocheck).
+
+To open it in Napari, remove any existing image layers, then drag and drop the 
+file over the canvas. Note this can take a while to open, so give it some time! 
+Alternatively, you can select in the top menu-bar:  
+`File > Open File(s)...`
+
+![](fig/cells-time-napari.png){alt="A screenshot of a 2D time series in Napari"}
+
+This image is a 2D time series (tyx) of some human cells undergoing mitosis. The 
+slider at the bottom now moves through time, rather than z or channels. Try 
+moving the slider from left to right -  you should see some nuclei divide and 
+the total number of nuclei increase.
+
+We can again check the image dimensions by running the following:
+
+```python
+image = viewer.layers["00001_01.ome"].data
+print(image.shape)
+```
+
+```output
+# (t, y, x)
+(93, 1024, 1344)
+```
+
+Note that this image has a total of 3 dimensions, and so will also be stored in 
+a 3D array:
+
+![](fig/tyx-array.png){alt="A diagram of a tyx image array" width='40%'}
+
+This makes the point that certain dimensions/axes aren't always in the same 
```

Comment:
Would it make more sense to say "This makes the point that the dimensions don't always represent the same quantities. For example a 3D image array with shape (512, 512, 512) could represent a zyx, cyx or tyx image.